aerospike/assets/configuration/spec.yaml

```
@@ -122,6 +122,12 @@ files:
           - dc_remote_ship_ok
           - dc_delete_ship_attempts    # renamed dc_ship_delete_success
     - template: instances/default
+  - template: logs
+    example:
+      - type: file
+        path: /var/log/aerospike/aerospike.log
+        source: spark
+        server: <SERVICE_NAME>
```

```suggestion
        source: aerospike
```
I think we prefer to let the user specify the `service` elsewhere